packages/base/src/commands.ts

```diff
@@ -320,7 +320,7 @@ export function addCommands(
 
   commands.addCommand(CommandIDs.buffer, {
     label: trans.__('Buffer'),
-    isEnabled: () => isLayerTypeSupported(tracker),
+    isEnabled: () => selectedLayerIsOfType(['VectorLayer'], tracker),
```

Comment:
```suggestion
    isEnabled: () => selectedLayerIsOfType(['VectorLayer', 'ShapefileLayer'], tracker),
```

Reply:
I realised I have been doing it wrong and for some reason TypeScript didn't complain too. https://github.com/geojupyter/jupytergis/blob/main/packages/schema/src/schema/project/jgis.json#L30 `VectorLayer` shall cover shapefile too

Reply:
Ah right thanks for the clarification!